.circleci/config.yml

```diff
@@ -764,26 +755,131 @@ jobs:
               fi
             fi
 
+  test_deploy_pypi:
+    machine:
+      image: circleci/classic:201711-01
+    working_directory: /tmp/src/fmriprep
+    steps:
+      - checkout
+      - run:
+          name: Build fMRIPrep
+          command: |
+            pyenv local 3.5.2
+            pip install twine  # For use in checking distributions
+            THISVERSION=$( python get_version.py )
+            virtualenv --python=python build
+            source build/bin/activate
+            pip install --upgrade "pip>=19.1" numpy
+            echo "${CIRCLE_TAG:-$THISVERSION}" > fmriprep/VERSION
+            python setup.py sdist
+            pip wheel --no-deps -w dist/ .
```

Comment:
Note that because of the CPython bits, this produces a wheel like `fmriprep-1.3.2+166.gbe8452ab-cp35-cp35m-linux_x86_64.whl`, which will only be installable with Python 3.5. We may want to consider distributing 3.6 and 3.7 wheels as well.

Reply:
Let's get this merged first :+1:

Reply:
Cool. Just stumbled down a little manylinux rabbit hole and wonder if it might make sense to pull the cpython bits out into a very small package that can be pre-compiled and that can let us drop the numpy/cpython build requirements for fMRIPrep.